src/main/java/edu/kis/powp/jobs2d/drivers/WorkspaceManager.java

```diff
@@ -11,21 +11,35 @@
  * This class allows setting a {@link CanvaShape} to represent the workspace area,
  * and automatically renders its border using a special {@link Job2dDriver}.
  */
-public class WorkspaceManager {
+public class WorkspaceManager implements Job2dDriver {
```

Comment:
WorkspaceManager is not the best name, it does not explain the purpose of this class.